### DISCUSSION TITLE
优选后的ip延迟还是很高，放到surge里延迟更高

### DISCUSSION THREAD

**Discussion by WEAKwind** [vvbbnn00/WARP-Clash-API#107] [Q&A]
这个是优选过的结果
<img width="938" alt="image" src="https://github.com/vvbbnn00/WARP-Clash-API/assets/37976540/081ff2e7-6cea-445d-9736-cb33687ab60c">
这个是surge里的延迟
<img width="783" alt="image" src="https://github.com/vvbbnn00/WARP-Clash-API/assets/37976540/d21a3299-4121-4018-9f7d-fdaee8b74164">

**Comment by cu3198** [comment]
你这个延迟是正常的吧，我遇到更离谱的，优选延迟一百多点，但是导入到clash之后测试延迟一千多，速度只有1Mb/s。
不清楚是什么情况，这个最终的延迟和速度跟什么有关呢？
我是在hyperv虚拟机的docker里面部署的。
